### ISSUE TITLE
Fresh conda-forge based install unable to import samgeo.

### ISSUE THREAD

**Issue by Lvulis** [opengeos/segment-geospatial#76]
### Environment Information

-   samgeo version: 0.7.0
-   Python version: 3.10.11
-   Operating System:  MS Windows Version 22H2 (OS Build 22621.1702)
-   Visual Studio Code: Version 1.78.2

### Description

I want to run `samgeo` for some planet data. I did a fresh install using conda-forge and mamba. I'm using the [input prompt example as a guideline.](https://samgeo.gishub.org/examples/input_prompts/). This fresh install is failing to call rasterio due to a DLL failure, which I *think* is related to a rasterio-gdal conflict (conda-forge/rasterio-feedstock#240). Maybe related to #53.

I tried the pip install version but got the MS Visual Studio C++ error documented in the [QGIS-SAM plugin FAQ](https://github.com/BjornNyberg/Geometric-Attributes-Toolbox/wiki/FAQ#failed-to-install-segment-geospatial) and don't want to install the 9 GB VS C++ development kit if I can avoid it. 

P.S. GREAT WORK!

### What I Did
In command line:
```
`mamba create -n samgeo segment-geospatial python leafmap -c conda-forge`
```
In visual studio code:
```python
import os
import leafmap
from samgeo import SamGeo, tms_to_geotiff

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Untitled-1 in line 4
      [2](untitled:Untitled-1?line=1) import os
      [3](untitled:Untitled-1?line=2) import leafmap
----> [4](untitled:Untitled-1?line=3) from samgeo import SamGeo, tms_to_geotiff

File [~\.conda\envs\samgeo\lib\site-packages\samgeo\__init__.py:8](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/__init__.py:8)
      [4](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/__init__.py?line=3) __email__ = 'giswqs@gmail.com'
      [5](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/__init__.py?line=4) __version__ = '0.7.0'
----> [8](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/__init__.py?line=7) from .samgeo import *

File [~\.conda\envs\samgeo\lib\site-packages\samgeo\samgeo.py:11](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/samgeo.py:11)
      [8](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/samgeo.py?line=7) import numpy as np
      [9](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/samgeo.py?line=8) from segment_anything import sam_model_registry, SamAutomaticMaskGenerator, SamPredictor
---> [11](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/samgeo.py?line=10) from .common import *
     [14](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/samgeo.py?line=13) class SamGeo:
     [15](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/samgeo.py?line=14)     """The main class for segmenting geospatial data with the Segment Anything Model (SAM). See
     [16](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/samgeo.py?line=15)     https://github.com/facebookresearch/segment-anything for details.
     [17](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/samgeo.py?line=16)     """

File [~\.conda\envs\samgeo\lib\site-packages\samgeo\common.py:13](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/common.py:13)
     [11](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/common.py?line=10) import shapely
     [12](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/common.py?line=11) import pyproj
---> [13](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/common.py?line=12) import rasterio
     [14](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/common.py?line=13) import geopandas as gpd
     [15](file:///~/.conda/envs/samgeo/lib/site-packages/samgeo/common.py?line=14) import matplotlib.pyplot as plt

File [~\.conda\envs\samgeo\lib\site-packages\rasterio\__init__.py:28](file:///~/.conda/envs/samgeo/lib/site-packages/rasterio/__init__.py:28)
     [24](file:///~/.conda/envs/samgeo/lib/site-packages/rasterio/__init__.py?line=23)                     os.add_dll_directory(os.path.abspath(p))
     [27](file:///~/.conda/envs/samgeo/lib/site-packages/rasterio/__init__.py?line=26) from rasterio._show_versions import show_versions
---> [28](file:///~/.conda/envs/samgeo/lib/site-packages/rasterio/__init__.py?line=27) from rasterio._version import gdal_version, get_geos_version, get_proj_version
     [29](file:///~/.conda/envs/samgeo/lib/site-packages/rasterio/__init__.py?line=28) from rasterio.crs import CRS
     [30](file:///~/.conda/envs/samgeo/lib/site-packages/rasterio/__init__.py?line=29) from rasterio.drivers import driver_from_extension, is_blacklisted

ImportError: DLL load failed while importing _version: The specified procedure could not be found.
```

**Comment by Lvulis** [comment]
Just fixed this by pinning the python version to 3.9 based on #53. Can now load/import `samgeo`. Recommend updating [docs](https://samgeo.gishub.org/installation/) on installation to pin the conda-forge installation to Python 3.9. Can PR it if you like. 
**What I did**
```python
mamba create -n samgeo segment-geospatial python==3.9 leafmap -c conda-forge
```
**What new docs should be**
```python
conda install -n base mamba -c conda-forge
mamba create -n geo segment-geospatial python==3.9 -c conda-forge```

**Comment by giswqs** [comment]
Thanks for reporting. This is probably an issue specific to Windows. The CI seems working fine for Python 3.9-3.11 on Linux. https://github.com/opengeos/segment-geospatial/actions/workflows/ubuntu.yml

**Comment by giswqs** [comment]
Fixed in #77
